src/controllers/institutes.js

```diff
@@ -61,29 +55,25 @@ module.exports.showInstitute = async (req, res, next) => {
     completed: true,
   });
   return res.json({
-    success: true, institute, activeEvents, completedEvents,
+    success: true,
+    institute,
+    activeEvents,
+    completedEvents,
   });
 };
 
 module.exports.editInstitute = async (req, res, next) => {
-  const {
-    name,
-    instituteId,
-    about,
-    externalUrl,
-    emailRegex,
-  } = req.body;
-  if (req.user.userType === 'mod') {
-    const instituteCount = await Institute.count(
-      {
-        instituteId: req.params.instituteId,
-        $in: { mods: req.user._id },
-      },
-    );
+  const { name, instituteId, about, externalUrl, emailRegex } = req.body;
```

Comment:
the previous code is much more readable and is recommended as per the styling guide, infact this is how it converts to when we run `npm run lint` or use vsc extensions for the same.